multidim-interop/versions.ts

```diff
@@ -33,13 +34,6 @@ export type Version = {
 }
 
 export const versions: Array<Version> = [
-    {
-        id: "rust-v0.48.0",
```

Comment:
Do you also need to remove the import in L8?

Reply:
Good catch. Fixed with https://github.com/libp2p/test-plans/pull/206/commits/dc85d3a93ab1e8b38421e863c00458e386f9ecff.